apps/opik-backend/config.yml

```diff
@@ -49,6 +49,7 @@ health:
 
 distributedLock:
   lockTimeoutMS: ${DISTRIBUTED_LOCK_TIME_OUT:-500}
+  bulkLockTimeoutMS: ${DISTRIBUTED_BULK_LOCK_TIME_OUT:-5000}
```

Comment:
You should clean up all the bulk lock code as it's no longer used. It can be created at a later point if needed and in a separated PR. 

The reason is that we want people to avoid using a lock strategy at all cost. That should be the last resort to solve a concurrency problem and only when there's no other choice. We should review and pick up those cases carefully.

Reply:
I missed that, this shouldn't be need anymore